src/androidprojectsource.cpp

```diff
@@ -0,0 +1,49 @@
+/***************************************************************************
+  androidprojectsource.cpp - AndroidProjectSource
+
+ ---------------------
+ begin                : 19.3.2018
+ copyright            : (C) 2018 by Matthias Kuhn
+ email                : matthias@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsmessagelog.h"
+#include "androidprojectsource.h"
+#include <QFile>
+#include <QtAndroid>
```

Comment:
The code has been restructured a bit.
These files will need to go to `src/core`

Reply:
... and probably listed in core.pro

Reply:
yes, I saw, thank you. They had not been merged by git...

Reply:
Careful blaming git, it will jinx you